05_syntax_lab/02.py

```diff
@@ -1,6 +1,10 @@
-"""
-Write a Python program that randomizes 7 numbers
-and prints their sum total.
-If the sum is divisable by 7, also print the word "Boom"
-"""
-
+from random import randint
+    
+sum = 0
+for x in range(1,7) :
+    num = randint(1,100)
+    sum += num
+
+print sum
+if (sum % 7) == 0 :
+    print "BOOM"
```

Comment:
range in python includes the low number but not the high number so range(1, 7) will cover 1,2,3,4,5,6.